frontend/src/specialButtons/CloseButton.js

```diff
@@ -1,8 +1,13 @@
 import React from 'react'
-import { XSquareFill } from 'react-bootstrap-icons'
+import { FileLockFill } from 'react-bootstrap-icons'
 
-function CloseButton({ onClick }) {
-  return <XSquareFill onClick={onClick} className="editIcon closeIcon" />
+function CloseButton({ onClick, hidden }) {
+  return (
+    <FileLockFill
+      onClick={onClick}
+      className={'editIcon closeIcon' + (hidden ? ' hidden' : '')}
```

Comment:
Template stringet használjunk inkább ;)

Reply:
Nem tudtam, hogy kifejezést is lehet írni a template-be. Javítva.